src/builder/QuarkBuilder.sol

```diff
@@ -9,10 +9,11 @@ import {BridgeRoutes} from "./BridgeRoutes.sol";
 import {EIP712Helper} from "./EIP712Helper.sol";
 import {Strings} from "./Strings.sol";
 import {PaycallWrapper} from "./PaycallWrapper.sol";
+import {QuarkBuilderMath} from "./QuarkBuilderMath.sol";
 import {QuotecallWrapper} from "./QuotecallWrapper.sol";
 import {PaymentInfo} from "./PaymentInfo.sol";
 
-contract QuarkBuilder {
+contract QuarkBuilder is QuarkBuilderMath {
```

Comment:
We have been using the internal library pattern. So you can define `QuarkBuilderMath` as a library with private functions and not have to inherit it from `QuarkBuilder`.

Reply:
deleted QuarkBuilderMath; just using Hans' Math contract